### DISCUSSION TITLE
Create new patch "Custom template(s)"

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#198] [Feature Requests] [answered]
Would it make any sense to open up the possibility to have custom templates?
I find myself starting almost all of my patches with more or less the same ops. Recreating these over and over and over again, it made my thinking of this possibility.
In most other software I use, there is a way to save a version for "use at start up". I personally use this heavily so I wanted to start a discussion to find out if this is useful for others as well.

**Comment by steam0r** [comment, marked as answer]
this could be easily done by creating these patches and then "cloning" them, right? simply create your custom template and then start from there every time you create a new patch. by simply using "save as".

**Reply by franckaubry** [reply]
If this would be a useful feature for others, I was thinking that users could have a little more control over which templates show up on the list, in what order and also a way to promote a patch to become a template.

Your workflow @steam0r is absolutely a way to go if I am the only one that sees this need!
I can just give myself a "Template" namespace and start filling up different templates there. :)

**Reply by steam0r** [reply]
you could also create a "team" (maybe even for you alone) and collect patches there. you could then share these with collaborators. with the next release team-members will also always be able to open the patch in the editor read-only, so these could then be cloned by them.
